sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuColumnarToRowExec.scala

```diff
@@ -293,13 +298,23 @@ abstract class GpuColumnarToRowExecParent(child: SparkPlan, val exportColumnarRd
     val f = makeIteratorFunc(child.output, numOutputRows, numInputBatches, opTime, collectTime)
 
     val cdata = child.executeColumnar()
-    if (exportColumnarRdd) {
+    val rdata = if (exportColumnarRdd) {
       // If we are exporting columnar rdd we need an easy way for the code that walks the
       // RDDs to know where the columnar to row transition is happening.
       GpuColumnToRowMapPartitionsRDD.mapPartitions(cdata, f)
     } else {
       cdata.mapPartitions(f)
     }
+
+    postTransition match {
+      case Some(transformations) =>
```

Comment:
So what happens if transformations is empty?  Would it be cleaner to just have transformations not be optional but just be a Sequence, and go off of an empty sequence instead of None?